MARKETING/PressReleaseDrafts/2023/draft.md

```diff
@@ -53,13 +53,12 @@ Many [software packages and open-source projects now support WoT](https://www.w3
 tools for Thing Description validation and construction as well as support for Thing runtimes and 
 Thing Description directories supporting discovery.
 
-Market adoption of the Web of Things is rapidly growing.
-The Web of Things has been adopted or is under consideration for adoption by
-several standards organizations, including 
+The market adoption of the Web of Things is gaining momentum, with notable products such as SayWoT! and Desigo CC from Siemens, the CGLL Platform from Takenaka, <some more>.
```

Comment:
looks good to me.

@koalie what do you think?

Reply:
From an active IE. In my understanding, the mentioning of market adoptions can also include products from non W3C members.

Reply:
OK. I reserve the right to re-assess once the draft has stabilized and if needed (because the press release is too long, or for other reasons) to elide.

Reply:
If you would be open to it, I would be very happy to mention Krellian here as another example of commercial adoption (from another Invited Expert in the startup world).

```suggestion
Market adoption of the Web of Things is gaining momentum, with notable products such as SayWoT! and Desigo CC from Siemens, the CGLL Platform from Takenaka, and interesting products from the startup world like MONAS and Agorà from VAIMEE systems for creating digital twins of cast-resign transformers and crops-terrains, and Krellian Cloud which provides real-time data analytics for smart buildings.
```

This is in reference to [Krellian Cloud](https://krellian.com/cloud/), which has recently launched in private alpha following an [Innovate UK](https://www.ukri.org/councils/innovate-uk/) funded project.

If it's getting too long then you could just mention the products by name.

Reply:
would be also ok, from my side. @koalie what do you think?

Reply:
Yes, let's try with this and if needed / as needed, we can trim down one this is being reviewed by Members.